src/AttachmentAction.php

```diff
@@ -152,6 +159,26 @@ public function setValue(?string $value): self
         return $this;
     }
 
+    /**
+     * @return string|null
```

Comment:
You can remove this, strict typing already give the information.

Reply:
I wanted to maintain the exact same styling as the whole class, but I can remove this if you want.

Reply:
Yeah I understand, the code was created before PHP 7. So unless the CI bot is crying, you can remove it. :+1:

Reply:
Then it should be removed from the whole file, otherwise there will be no consistency.

Reply:
Consistency is not a very big deal for this, but anyway let's keep it that way.

If you want to clean files up on this repo next, feel free! :+1: